src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -369,6 +372,14 @@ void Convolution::getSupportedDescriptors() {
                  dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core) && !canBeExecutedInInt8() &&
                  getParentEdgeAt(1)->getParent()->isConstant() && getParentEdgeAt(1)->getParent()->getType() == Type::Input &&
                  (withBiases ? (getParentEdgeAt(2)->getParent()->isConstant() && getParentEdgeAt(2)->getParent()->getType() == Type::Input) : true);
+
+        if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core)) {
```

Comment:
Please add the comment that explains the logic: smt about AVX512 brconv is disabled by default due to performance issues but user has an ability to force it via Primitives priority mechanism

Reply:
Added.